src/modelio.cs

```diff
@@ -1771,8 +1791,13 @@ interface MDLTextureSampler
 
 	[iOS (9,0), Mac(10,11, onlyOn64 : true)]
 	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor] // designated
 	interface MDLTransform : MDLTransformComponent, NSCopying {
 
+		[DesignatedInitializer]
+		[Export ("init")]
+		IntPtr Constructor ();
```

Comment:
`[DesignatedDefaultCtor]` was added last year

Reply:
Missed it, thanks (: